.github/workflows/ci.yml

```diff
@@ -15,3 +15,6 @@ jobs:
   call-terraform-ci-pipeline:
     uses: terraform-ibm-modules/common-pipeline-assets/.github/workflows/common-terraform-module-ci.yml@v1.7.3
     secrets: inherit
+    with:
+      craTarget: "examples/basic"
```

Comment:
`examples/basic` is probably the wrong directory to scan here. We should probably be scanning `patterns/roks`. We may also need to scan `patterns/vsi` but I don't think our pipeline currently supports scanning multiple directories, so maybe go with roks for now

Reply:
Made the required changes